en/deployment/articles/02-configuring-liferay/04-liferay-portal-clustering/05-enabling-cluster-link.markdown

```diff
@@ -130,87 +130,91 @@ from using UDP Multicast clustering, you can configure TCP Unicast. You must use
-    to a location accessible to @product@.
+4.  Using separate control and transport channel ports lets you monitor control and transport traffic and helps you isolate information to diagnose problems. 
 
-4.  Make a copy of the `tcp.xml` in the same location and rename both files, 
-    designating one for the control channel and the other for the transport
-    channel. For example,
+    Make a copy of the `tcp.xml` in the same location and rename both files, designating one for the control channel and the other for the transport channel. For example, you could use these file names:
 
     -   `tcp-control.xml`
 
     -   `tcp-transport.xml`
 
-    Copy these two files to all your nodes. 
-
-5.  If you're vertically clustering (i.e., you have multiple servers running on
-    the same physical or virtual system), every channel must use a unique unused
-    bind port for discovery communication. In each `tcp-*.xml` file, assign the
-    TCP tag's `bind_port` attribute to a unique, unused port.  
+5.  Modify the [Cluster Link properties](https://docs.liferay.com/portal/7.2-latest/propertiesdoc/portal.properties.html#Cluster%20Link) in the node's `portal-ext.properties` file to enable Cluster Link and point to the TCP XML file for each Cluster Link channel: 
 
-    ```xml
-    <TCP bind_port="[some unused port]"
-        ... 
-    />
+    ```properties
+    cluster.link.enabled=true
+    cluster.link.channel.properties.control=/ehcache/tcp-control.xml
+    cluster.link.channel.properties.transport.0=/ehcache/tcp-transport.xml
     ```
 
+6.  Modify each `tcp-*.xml` file's the TCP and TCPPing elements to account for each node's IP address and bind port. 
+
+    If you're vertically clustering (i.e., you have multiple servers running on the same physical or virtual system), every channel must use a unique unused bind port for discovery communication. In each `tcp-*.xml` file, assign the TCP tag's `bind_port` attribute to a unique, unused port.
+
     For example, your first two nodes might assign these bind ports:
 
-    **Node 1**
+    | Node   | Properties File     | Port   |
+    | :----- | :------------------ | :----- |
+    | Node 1 | `tcp-control.xml`   | `7800` |
+    | Node 1 | `tcp-transport.xml` | `7801` |
+    | Node 2 | `tcp-control.xml`   | `7802` |
+    | Node 2 | `tcp-transport.xml` | `7803` |
 
-    -   `tcp-control.xml` bind port: `7800`
-    -   `tcp-transport.xml` bind port: `7801`
+    Here are example TCP and TCPPing elements using the bind ports on nodes running on the same system (i.e., same IP address):
 
-    **Node 2**
+    **Node 1 `tcp-control.xml`**
 
-    -   `tcp-control.xml` bind port: `7802`
-    -   `tcp-transport.xml` bind port: `7803`
+    ```xml 
+    <TCP bind_port="7800"/> 
+    <TCPPING async_discovery="true"
+        initial_hosts="192.168.224.154[7800],192.168.224.154[7802]"
+        port_range="1"/> 
```

Comment:
@jhinkey Is `port_range="1"` intended here? The range means JGroups will try to probe the host starting at the `port number`, and ending at `port number + port_range`.

> Example: initial_hosts=A[7800] port_range=0 probes A:7800, port_range=1 probes A:7800 and A:7801

Here we are setting exact accurate host information, so I think `port_range` should just be `0`.

cc @tinatian

Reply:
@dantewang Yes, my intention was to set exact host info. I'll set the `port_range="0"`. Thanks.

Reply:
Set the port range to 0 in https://github.com/tinatian/liferay-docs/pull/8/commits/12e631090ffd784dd1b8f426a04d7b72fe63c750